package.json

```diff
@@ -17,7 +17,7 @@
     "@types/react-dom": "^17.0.0",
     "contentful-ui-extensions-sdk": "^3.31.0",
     "cross-env": "^7.0.3",
-    "imgix-management-js": "^1.1.0",
+    "imgix-management-js": "^1.2.1-beta.1",
```

Comment:
My spidey senses are tingling here, but I'm guessing we're going to upgrade this to a stable version before releasing this library, yah?

Reply:
That's correct 👍